project_templates/stack_package/README.md

```diff
@@ -1,38 +1,267 @@
+
+Example: [bin.src/SConscript](example/bin.src/SConscript).
+
+sconsUtils/scons use this `SConscript` file.
+You shouldn't need to modify this file.
+
+The `bin.src/` directory is where you put Python scripts (including command-line task executables) that are installed for users.
+
+### doc/{{cookiecutter.package_name}}/index.rst
+
+Example: [doc/example/index.rst](example/doc/example/index.rst).
+
+This is the main documentation page for the *package* (as opposed from the Python module, see below).
+The purpose of this page is to document the Git repository itself, and anything that exists outside the normal codebase.
+An example use of this package index page (or pages linked from it) is to document datasets in a Git LFS-based data package.
+
+At a minimum, this page should be customized with a description of the package.
+This copy can be shared with the description in the README.
+
+### doc/{{cookiecutter.python_module}}/index.rst
+
+Example: [doc/lsst.example/index.rst](example/doc/lsst.example/index.rst).
+
+This is the main documentation page for the Python module provided by the package.
+Most documentation will be linked through this page.
+Packages that provide multiple Python modules (like ``afw``) can have several of these directories, each named after the public module's namespace.
+
+Separate documentation is forthcoming on how to format this page.
+
+### doc/conf.py
+
+Example: [doc/conf.py](example/doc/conf.py).
+
+This `conf.py` file is only used for single-package documentation builds during development.
+You shouldn't modify this file beyond the basic templated customization.
+If you need to modify the Sphinx configuration, post a message in the #dm-docs Slack channel so it can be included in [documenteer](https://github.com/lsst-sqre/documenteer).
+
+### doc/doxygen.conf.in
+
+Example: [doc/doxygen.conf.in](example/doc/doxygen.conf.in).
+
+This file configures the Doxygen build.
+Per package Doxygen builds are necessary to generate the C++ API reference.
+
+### doc/index.rst
+
+Example: [doc/index.rst](example/doc/index.rst).
+
+This file is the root page for single-package documentation builds.
+It is not used at all for the published site (https://pipelines.lsst.io).
+This page's `toctree` should link to the `index.rst` files at the root of both the package and Python module documentation directories.
+
+### doc/manifest.yaml
+
+Example: [doc/manifest.yaml](example/doc/manifest.yaml).
+
+This YAML file is used by the Sphinx documentation build tool (included in the [documenteer](https://github.com/lsst-sqre/documenteer) package).
+It defines what package and Python module documentation directories this package provides.
+The Cookiecutter template should configure this file for you out-of-the box.
+
+Separate documentation is forthcoming on how to use this file.
+
+### doc/SConscript
+
+Example: [doc/SConscript](example/doc/SConscript).
+
+sconsUtils/scons use this `SConscript` file for the Doxygen build.
```

Comment:
This wording is a little weird to me.  Are you saying "sconsUtils and scons use ..." or "scons (extended by sconsUtils) uses ...".  I think it's actually the second.

This wording is used in several places.